src/Sulu/Bundle/ContactBundle/DependencyInjection/SuluContactExtension.php

```diff
@@ -70,6 +70,24 @@ public function prepend(ContainerBuilder $container)
                 ]
             );
         }
+
+        if ($container->hasExtension('sulu_admin')) {
```

Comment:
That's what I've already meant before: The `sulu_admin` needs to be here anyway, so even an error would be better than having the code in the block not being executed...

Reply:
?? what